compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -6,6 +6,7 @@
 import logging
 import pathlib
 
+import jinja2
 import yaml
```

Comment:
I'd like to start thinking about how we can pare down the memory requirements.  Thinking from an administrator's perspective, there will potentially be hundreds of user endpoints running on the head node at any given moment, and we are not currently the smallest tool.

Suggestion: don't importing `jinja2` here, but rather in the method that uses it.  In particular, the rendering logic is only used by the endpoint _manager_, prior to the `exec()`.  I see no value in uselessly cluttering the child's memory with "yet another" library, that, in this case, is never utilized.